vignettes/GOstatsHyperG.Rmd

```diff
@@ -0,0 +1,512 @@
+---
+title: "How To Use GOstats Testing Gene Lists for GO Term Association"
+author:
+  - name: S. Falcon 
+  - name: R. Gentleman
+  - name: Dennis Ndubi
+    affiliation: "Vignette translation from Sweave to R Markdown / HTML"
+date: March 15, 2022
+package: GOstats
+bibliography: GOstats.bib
+vignette: >
+  %\VignetteIndexEntry{How To Use GOstats Testing Gene Lists for GO Term Association}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
```

Comment:
The vignette entry should follow those from the Rnw as closely as possible:  

```
%\VignetteIndexEntry{Hypergeometric Tests Using GOstats}
%\VignetteDepends{ALL, GO.db, annotate, hgu95av2.db, genefilter, RColorBrewer,
%xtable, Rgraphviz} 
%\VignetteKeywords{Hypergeometric, GO, ontology}
%\VignettePackage{GOstats}
```
However, keep 
```
 %\VignetteEngine{knitr::rmarkdown}
%\VignetteEncoding{UTF-8}
```